TypeCobol/Parser.cs

```diff
@@ -145,9 +148,20 @@ public IEnumerable<CodeElement> CodeElements
 		}
 
 
-
-		public static Parser Parse(string path, DocumentFormat format, bool autoRemarks = false, IList<string> copies = null, IAnalyzerProvider analyzerProvider = null) {
-			var parser = new Parser();
+#if EUROINFO_RULES
+        public static Parser EIParse(string path, DocumentFormat format, bool autoRemarks = false, IList<string> copies = null, IAnalyzerProvider analyzerProvider = null, string cpyCopyNamesFile = null)
+        {
+            var parser = new Parser();
+            var typeCobolOption = new TypeCobolOptions() { ExecToStep = ExecutionStep.Generate };
+            typeCobolOption.AutoRemarksEnable = autoRemarks;
+            typeCobolOption.CpyCopyNamesMapFilePath = cpyCopyNamesFile;
+            parser.Init(path, typeCobolOption, format, copies, analyzerProvider);
+            parser.Parse(path);
+            return parser;
+        }
+#endif
```

Comment:
In order to avoid a method specific to EUROINFO_RULES can you use a more generic signature like this:
```suggestion
        public static Parser Parse(string path, DocumentFormat format, IList<string> copies = null, IAnalyzerProvider analyzerProvider = null, TypeCobolOptions typeCobolOption)
        {
            ...
        }
#endif
```

Reply:
I agree we should rely more on `TypeCobolOptions` class to pass options in different test cases otherwise the signatures will have to change frequently.

Reply:
done